doc/tutorial/Authentication.lhs

```diff
@@ -281,17 +281,22 @@ lookupAccount key = case Map.lookup key database of
 For generalized authentication, servant exposes the `AuthHandler` type,
 which is used to wrap the `Request -> Handler user` logic. Let's
 create a value of type `AuthHandler Request Account` using the above `lookupAccount`
-method:
+method (note: we depend upon `cookie`'s `parseCookies` for this):
```

Comment:
I don't know of a way to check before it's merged. Once it's merged, you can go to http://haskell-servant.readthedocs.io/ and click "v: stable" at the bottom and pick *master* instead.